single-node-k8s.sh

```diff
@@ -135,6 +135,12 @@ then
     ./podman2minikube.sh "rook/ceph:${ROOK_VERSION}"
 fi
 
+# Rook also uses ceph/ceph:v15 (build.env:BASE_IMAGE), so push it into the VM
+if [ -n "${BASE_IMAGE}" ] && podman inspect "${BASE_IMAGE}" > /dev/null
```

Comment:
Rook not expected to use the base image that cephcsi uses. rook uses different tag of ceph images see https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/cluster.yaml#L24

Reply:
That actually gets replaced in `scripts/rook.sh`. But yes, it seems that uses `ROOK_CEPH_CLUSTER_VERSION` and not `BASE_IMAGE`. Which is a little weird, as the logs mentions pulling `ceph/ceph:v15` failed.

This might need more investigation as of why the wrong image gets configured/pulled, but for now this will work.

Reply:
let's keep track of it

Reply:
Added as a checkbox to #1693